README.md

```diff
@@ -359,17 +373,37 @@ Please execute at least step 1 + 2 if before importing restdocs-wiremock into yo
 
 ## Publishing
 
-This project makes use of the [axion-release-plugin](https://github.com/allegro/axion-release-plugin)
-and publishing is automated in travis, when a new release is tagged in git.
-
-Locally you should be able to create a new release by running the `release` task on gradle. A successful
-travis build of this tag should finally end up on [bintray](https://bintray.com/epages/maven/restdocs-wiremock/).
+Given that the `master` branch on the upstream repository is in the state from which you want to create a release, execute the following steps:
 
 ```shell
-./gradlew clean build release
+git checkout master
+git pull upstream master
+
+# Print current version to the terminal
+./gradlew currentVersion
+
+# (a) Release new patch version
+./gradlew release -Prelease.versionIncrementer=incrementPatch
+
+# (b) Release new minor version
+./gradlew release -Prelease.versionIncrementer=incrementMajor
+
+## (c) Release new major version
+./gradlew release -Prelease.versionIncrementer=incrementMajor
```

Comment:
> Once this PR is merged I want to create a release with the version number "0.8.0". This should trigger a Travis build and create staging repository at Sonatype. This I want to manually close and release which should make restdocs-wiremock available on Maven Central.

Do you intend to create the release using GitHub UI? If so and if this works, I hope those steps shouldn't be necessary.

Reply:
No, as far as I understand calling this Gradle task will also create the release entry on GitHub.

Let's see how it works. We can later discuss the pros and cons of using this Gradle plugin.

Reply:
I had the following problems with the `release` task.

1. I mixed up the terms "major" version and "minor" version (both in the description in the README file and when actually calling the task)
2. The plugin also tries to update the version number in the README file. So I cannot create a release independently due to the branch protection
3. The plugin tries to push to the `origin` repository which points to my fork. So some additional configuration is needed.

All in all, there are many things one needs to understand when using the tool. So I am now also convinced that a handful of manual steps for creating a release are better than using this plugin.